README.md

```diff
@@ -429,17 +429,37 @@ misbehavingStream.log();
 
 ### Future
 
-#### `Future#listen(o: Consumer<A>): void`
+#### `Future.of(a: A): Future<A>`
 
-Adds a consumer as listener to a future. If the future has already
-occurred the consumer is immediately pushed to.
+Converts any value into a future that happens immediately.
 
 #### `fromPromise<A>(p: Promise<A>): Future<A>`
 
 Converts a promise to a future.
 
+#### `isFuture(f: any): f is Future<any>`
+
+Returns `true` if `f` is a future and `false` otherwise.
+
+#### `Future#listen(o: Consumer<A>): void`
+
+Adds a consumer as listener to a future. If the future has already
+occurred the consumer is immediately pushed to.
+
 ### Stream
 
+#### `empty: Stream<A>`
```

Comment:
If I recall correctly it has the type `Stream<any>`.

Reply:
Ah, sorry, I'm new to TypeScript (from Haskell) and to me those seem like the same type

Reply:
I see what you mean :smile:. I actually remember being slightly confused when I wrote `empty`. I tried to use a generic. But TypeScript only allows generics on functions and it turned out that `any` works best anyway.